src/System.Private.CoreLib/shared/System/Collections/Generic/List.cs

```diff
@@ -165,6 +165,8 @@ public int Capacity
             }
         }
 
+        internal Span<T> AsSpan() => new Span<T>(_items, 0, _size);
```

Comment:
Make `_items` and `_size` internal, and avoid adding a new method to `List<T>` ?

Reply:
Done